doc/TROUBLESHOOTING.md

```diff
@@ -166,7 +166,7 @@ ConversionException: Could not call com.cloudbees.jenkins.plugins.sshcredentials
 
 ### Selenium Grid agents failed to connect
 
-On recent versions of Jenkins Core, [Agent - Master Access Control](https://wiki.jenkins.io/display/JENKINS/Slave+To+Master+Access+Control) was introduced it seems causes an issue with Selenium Grid Agents, to fix this problem you have to disable: "Manage Jenkins" > "Configure Global Security", and check "Enable Agent → Master Access Control" (as it's said in Jenkin documentation)
+On recent versions of Jenkins Core, [Agent - Controller Access Control](https://wiki.jenkins.io/display/JENKINS/Slave+To+Master+Access+Control) was introduced it seems causes an issue with Selenium Grid Agents, to fix this problem you have to disable: "Manage Jenkins" > "Configure Global Security", and check "Enable Agent → Controller Access Control" (as it's said in Jenkin documentation)
```

Comment:
All of this is horrifying, would just comment out.

"Recent versions" referring to releases since late 2014. The advice to disable it. The lack of confidence in whether this is the cause.

Reply:
What do you suggest? To revert the change?

Reply:
I think Daniel wants you to remove the entire section :)

Reply:
Ok thanks. I removed the section :)